Version3/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Table Version 1</title>
+</head>
+<body id="main">
+  <table id="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Alter</th>
+        <th>Größe</th>
+        <th>Wohnort</th>
+        <th>Schule</th>
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+       <td>Kevin</td>
+       <td>15</td>
+       <td>165cm</td>
+       <td>Berlin</td>
+       <td>Gymnasium Berlin</td>
+      </tr>
+
+      <tr>
+        <td>Annabele</td>
+        <td>18</td>
+        <td>169cm</td>
+        <td>Berlin</td>
+        <td>Realschule Berlin</td>
+       </tr>
+     </tbody>
+    </table>
+  <script src="reziable.js"></script>
+
+  <script>
+  var table = document.getElementsByTagName('table')[0];
+  resizableGrid(table);
```

Comment:
```suggestion
    var table = document.getElementsByTagName('table')[0];
    resizableGrid(table);
```

Move this to its own file.